### DISCUSSION TITLE
Hikvision DS-2CD2543G0-IS - No video - No NVR

### DISCUSSION THREAD

**Discussion by ausburbs** [Sunoo/homebridge-camera-ffmpeg#1318] [Q&A]
Installed a new Hikvision DS-2CD2543G0-IS and followed most guidance here. Haven't seen a reference to this mode but should be similar to other Hikvision products. Using HomeBridge on a Synology NAS. I'm not using an NVR, just looking to access the live stream.

1. For "source" I've tried with and without "-rtsp_transport tcp -re -i"
2. Both source URLs work perfectly in VLC and display live feed. 
3. I've tried bridged and unbridged settings. 

The tile appears in HomeKit and displayed "No Response." 

Would love any insights!
 
{
            "name": "Camera FFmpeg",
            "cameras": [
                {
                    "name": "Driveway",
                    "unbridge": false,
                    "videoConfig": {
                        "source": "-i rtsp://uid here:passhere@10.0.0.51:554/Streaming/Channels/101",
                        "stillImageSource": "-i http://uid here:passhere@10.0.0.51:554/Streaming/Channels/102",
                        "maxWidth": 2688,
                        "maxHeight": 1520,
                        "maxBitrate": 6144,
                        "forceMax": true,
                        "audio": false
                    }
                }
            ],
            "platform": "Camera-ffmpeg"
        }

**Comment by Sunoo** [comment]
Can you turn on debug mode and share your log from when you try to access a camera?

**Comment by ausburbs** [comment]
[homebridge.log.txt](https://github.com/Sunoo/homebridge-camera-ffmpeg/files/9084655/homebridge.log.txt)

Enabled debug this morning and attempted to connect. Thank you in advance for your help!

**Reply by ausburbs** [reply]
@Sunoo any ideas looking at my code? Thanks!

**Reply by Sunoo** [reply]
Sorry, somehow didn’t notice your reply until now. I don’t see anything in your log though.
